source/administration/server-side-encryption.rst

```diff
@@ -51,10 +51,12 @@ supports the following encryption strategies:
       Deployment, see :ref:`minio-encryption-sse-kms-quickstart`. For
       production MinIO deployments, use one of the following guides:
 
-      - :ref:`AWS SecretsManager <minio-sse-aws>`
-      - :ref:`Google Cloud SecretManager <minio-sse-gcp>`
-      - :ref:`Azure Key Vault <minio-sse-azure>`
-      - :ref:`Hashicorp KeyVault <minio-sse-vault>`
+      - :kes-docs:`Azure Key Vault <integrations/azure-keyvault/>`
+      - :kes-docs:`AWS SecretsManager <integrations/aws-secrets-manager/>`
```

Comment:
Should this be `Secrets Manager` or `Secrets Manager`? The linked KES page says `Secrets Manager`

Reply:
Requires the space per Amazon. Fixed throughout.
Thanks for the catch!